packages/foundations/scss/_screen-sizes.scss

```
@@ -27,7 +27,7 @@ $db-breakpoint-xl: functions.to-rem(1920);
 	}
 
 	@if ($width == "min") {
-		@media screen and (min-width: $screen) {
+		@media screen and (width >= $screen) {
```

```suggestion
		@media screen and (min-width: $screen) {
```

The argument to leave this as it was is that it's downward compatible.

---

That's sad, it shouldn't break at all.

But I think we can skip it, it doesn't work on [IE](https://caniuse.com/css-media-range-syntax) but we don't support it anyways...

---

It's not just IE, it's even also some others, even though that all of these are under 1% market share:
https://caniuse.com/css-media-range-syntax

Nevertheless, if one of these might be more "resilient" than the other, I would have preferred that one.

---

fixed